Gems/PhysX/Code/Source/Joint/Configuration/PhysXJointConfiguration.cpp

```diff
@@ -35,6 +35,14 @@ namespace PhysX
 
     }
 
+    JointMotorProperties::JointMotorProperties(bool enabled, float gear_ratio, float force_limit)
```

Comment:
nit: Parameter names underscore naming -> `gearRatio`, `forceLimit`

Reply:
adjusted to code standard.